Lib/fontTools/feaLib/builder.py

```diff
@@ -764,12 +764,25 @@ def buildGDEFGlyphClassDef_(self):
             classes = {g: c for (g, (c, _)) in self.glyphClassDefs_.items()}
         else:
             classes = {}
+            ignoreClassFlagsUsed = False
             for lookup in self.lookups_:
+                ignoreClassFlagsUsed = (
+                    ignoreClassFlagsUsed or (
+                        lookup.lookupflag & (
+                            otl.LOOKUP_FLAG_IGNORE_BASE_GLYPHS +
+                            otl.LOOKUP_FLAG_IGNORE_LIGATURES +
+                            otl.LOOKUP_FLAG_IGNORE_MARKS
```

Comment:
so if _any_ of the `lookupflags Ignore*` are used, you infer all of them? I would have interpreted the wording of the FEA spec as meaning that, if `ignoreMarks` is used, infer class 3, if `ignoreLigature` is used, infer class 2, etc.

Reply:
(emphasis mine)

> if any of the lookup flags for skipping glyphs _of a certain class_...